### DISCUSSION TITLE
Windows binary snapshots

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#726] [Announcements]
We have recently introduced Windows snapshots, so the Windows user can get access to the latest build without need to compile the application. Here is a short instruction how to download snapshots. See also #719 The portable package doesn't include Ngspice. Use the following link to download the latest snapshots: https://github.com/ra3xdh/qucs_s/releases/tag/continuous_build

![image](https://github.com/ra3xdh/qucs_s/assets/4920080/1a5dbf54-af51-446a-9894-0df69253b50e)

**Comment by zergud** [comment]
why 3 build targets for windows 64 - clang64, minwg64, ucrt64..?
in my opinion this will only confuse users - which build to download.

**Comment by zergud** [comment]
i think we need change version from current stable in snapshot
for now it show 24.2.1 for CI snapshot!
may be 24.3-4354e2b?

**Comment by ra3xdh** [comment]
> why 3 build targets for windows 64 - clang64, minwg64, ucrt64..?

UCRT64 surely could be disabled because it uses the same compiler as GCC environment.
